src/app/custom/enrichers/enrichers-catalog.json

```diff
@@ -334,5 +342,13 @@
         "swagger": "https://openapi.services.istex.fr/?urls.primaryName=pdf-text%20-%20Transforme%20un%20PDF%20en%20texte%20%C3%A0%20partir%20de%20l%27URL#/pdf-text/post-v1-pdf2txt",
         "objectifTDM": "https://services.istex.fr/extraction-du-texte-epure-de-pdf/",
         "type": "pretraitement"
-    }
+    },
+    {
+        "id": "post-v1-tools",
+        "url": "https://terms-extraction.services.istex.fr/v1/tools/normalize",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/terms-extraction",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=terms-extraction%20-%20Extraction%20de%20termes#/terms-extraction/post-v1-tools-normalize",
+        "objectifTDM": "https://services.istex.fr/2022/01/12/normaliser-un-texte-ou-un-terme/",
+        "type": "metadata"
+    },
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Wrong syntax

‏
```suggestion
    }
```